tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -191,6 +196,33 @@
    */
   long numBytes();
 
+  /**
+   * Returns the String representation of elements stored in the tensor.
+   *
+   * @param options overrides the default configuration
+   * @return the String representation of the tensor elements
+   * @throws IllegalStateException if this is an operand of a graph
+   */
+  default String dataToString(ToStringOptions... options) {
```

Comment:
I don't think the use of a vararg to handle the optional presence of `options` is a good idea. Having a second method accepting no parameter would be better. 

We use vararg options in the op wrappers because we want to limit the number of methods that ending up in the `*Ops` classes, which is already more than a thousand. But here it's fine "duplicating" it.

Reply:
I agree with @karllessard